src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -103,7 +104,7 @@ public static AppDomain CreateDomain(string friendlyName)
 
         public int ExecuteAssembly(string assemblyFile) => ExecuteAssembly(assemblyFile, null);
 
-        public int ExecuteAssembly(string assemblyFile, string[] args)
+        public int ExecuteAssembly(string assemblyFile, string?[]? args)
```

Comment:
Just curios why elements of args need to be nullable

Reply:
These are the arguments that'll be passed to `Main`, and while it'd be a little strange for nulls to be passed through in the array, nothing prevents it to my knowledge.